web_src/less/_repository.less

```diff
@@ -158,6 +158,8 @@
         }
 
         &.labels {
+            max-height: 300px;
```

Comment:
Maybe we should use [`vh`](https://www.w3schools.com/cssref/css_units.asp) for this? (e.g. `max-size: 30vh`) (I'm not familiar with any limitations or caveats that may come with it, [backwards compatibility](https://caniuse.com/#feat=viewport-units), etc.).

Reply:
`max-height: calc(50vh - anchorheight - margin)` should be safe in all cases, assuming the positioning script is smart enough to anchor upwards when the button is on the end of the vertical viewport.

Reply:
I believe the author is reproducing behavior from `.repository.new.issue .comment.form .metas .filter.menu` which sets 300px.